app/service/file_svc.py

```diff
@@ -73,6 +73,9 @@ def __init__(self, plugins, exfil_dir):
             file_path = await self._walk_file_path('plugins/%s/%s' % (plugin, location), name)
             if file_path:
                 return plugin, file_path
+            file_path = await self._walk_file_path('plugins/%s/data/%s' % (plugin, location), name)
```

Comment:
iterate through list instead of calling twice.

also, shouldn't walk file path "walk" it and find in the sub directory?

Reply:
I think _walk_file_path maybe the issue here

Reply:
The issue is that data/plugins not a sub directory of the given path, it's another parallel directory. Unless I'm understanding you wrong

Reply:
Changed to iterate through a for loop. But it's still getting called twice, is this what you wanted?